oml/models/resnet.py

```diff
@@ -42,16 +47,17 @@ def __init__(
         strict_load: bool = True,
     ):
         """
-        If you want to load FC layer from checkpoint and keep it untouched,
-        set both hid_dim and out_dim equal to None.
 
         Args:
-            weights: path to weights or the special key to download pretrained checkpoint, use None to randomly initialize model's weights or 'default' to use the checkpoint pretrained on ImageNet
-            arch: different types of resnet, please, check self.constructors
-            normalise_features: if normalise features
-            gem_p: value of power in GEM pooling
-            remove_fc: if remove fully connected layer
-            strict_load:  if strict load from checkpoint
+            weights: Path to weights or a special key to download pretrained checkpoint, use ``None`` to randomly initialize
+             model's weights or ``default`` to use the checkpoint pretrained on ImageNet.
+             You can check the available pretrained checkpoints in ``self.pretrained_models``.
+            arch: Different types of ResNet, please, check ``self.constructors``
+            normalise_features: Set ``True`` to normalise output features
```

Comment:
Grammar checker suggests to use normali**z**e

Reply:
let's ignore it since we need to rename all the methods as well (this is a UK vs USA thing)